code/game/objects/items/tools/mining_tools.dm

```diff
@@ -98,6 +86,261 @@
 	drill_verb = "drilling"
 
 
+/obj/item/tool/pickaxe/plasmacutter
+	name = "plasma cutter"
+	icon_state = "plasma_cutter_on"
+	var/base_icon_state = "plasma_cutter_on"
+	item_state = "gun"
+	w_class = 3.0 //it is smaller than the pickaxe
+	force = 70.0
```

Comment:
This value would make it the strongest melee weapon available instead of a tool. Reduce this to a value not higher than that of an entrenching tool.